docs/modules.rst

```diff
@@ -0,0 +1,7 @@
+cbc_sdk
```

Comment:
Not sure why does this file gets generated. It's some kind of a TOC file but I don't get where it is used.